website/templates/emails/welcome.html.mako

```diff
@@ -11,35 +11,46 @@
 <br>
+
 <h4>Make your work citable</h4>
-Every project and file on the OSF has a permanent unique identifier, and every registration can be assigned a DOI.  Citations for public projects are generated automatically so that visitors can give you credit for your research.<br>
+Every project and file on the OSF has a permanent unique identifier, and every registration can be assigned a DOI.  Citations for public projects are generated automatically so that visitors can give you credit for your research. <a href="http://help.osf.io/m/sharing/l/524208-create-dois/?utm_source=notification&utm_medium=email&utm_campaign=welcome">Learn more.</a><br>
 <br>
+
 <h4>Measure your impact</h4>
-You can monitor traffic to your public projects and downloads of your public files.<br>
+You can monitor traffic to your public projects and downloads of your public files. <a href="http://help.osf.io/m/projects/l/524052-view-analytics/?utm_source=notification&utm_medium=email&utm_campaign=welcome">Discover analytics.</a><br>
 <br>
+
 <h4>Connect services that you use</h4>
-GitHub, Dropbox, Google Drive, Box, Dataverse, figshare, Amazon S3, ownCloud, Bitbucket, GitLab, OneDrive, Mendeley, Zotero.  Do you use any of these? Link the services that you use to your OSF projects so that all parts of your project are in one place.<br>
+OSF integrates with GitHub, Dropbox, Google Drive, Box, Dataverse, figshare, Amazon S3, ownCloud, Bitbucket, GitLab, OneDrive, Mendeley, and Zotero. Do you use any of these? Link the services that you use to your OSF projects so that all parts of your research project are in one place. <a href="http://help.osf.io/m/addons/?utm_source=notification&utm_medium=email&utm_campaign=welcome">Learn about add-ons.</a><br>
 <br>
+
 <h4>Collaborate</h4>
-Add your collaborators to have a shared environment for maintaining your research materials and data and never lose files again.<br>
+Add your collaborators to have a shared environment for maintaining your research materials and data and never lose files again. <a href="http://help.osf.io/m/collaborating/?utm_source=notification&utm_medium=email&utm_campaign=welcome">Start collaborating.</a><br>
 <br>
-<br>
-Learn more about the OSF by reading the <a href="http://help.osf.io/">Guides</a>, or email <a href="mailto:${osf_contact_email}">${osf_contact_email}</a> with questions for support.<br>
+
+Learn more about the OSF by reading the <a href="http://help.osf.io/?utm_source=notification&utm_medium=email&utm_campaign=welcome">Guides</a>, or email <a href="mailto:${osf_support_email}">${osf_support_email}</a> for support.<br>
```

Comment:
"Learn more about the OSF by reading the"
should be 
"Learn more about OSF by reading the"